versioned_docs/version-7.x/getting-started.md

```diff
@@ -157,7 +157,7 @@ export default function App() {
 
 :::warning
 
-In a typical React Native app, the `NavigationContainer` should be only used once in your app at the root. You shouldn't nest multiple `NavigationContainer`s unless you have a specific use case for them.
+In a typical React Native app, the `NavigationContainer` should be only used once in your app at the root. You shouldn't nest multiple `NavigationContainers` unless you have a specific use case for them.
```

Comment:
`NavigationContainer` is the name of the component - so it's inside backticks. `NavigationContainer`s is intentional.